src/PKSim.Core/Model/ParameterRange.cs

```diff
@@ -86,71 +92,35 @@ public override string ToString()
 
       private static class AllRules
       {
-         public static IBusinessRule MinLessThanMax
-         {
-            get
-            {
-               return CreateRule.For<ParameterRange>()
-                  .Property(item => item.MinValueInDisplayUnit)
-                  .WithRule(minLessThanMax)
-                  .WithError((param, value) => PKSimConstants.Rules.Parameter.MinLessThanMax(param.ParameterName));
-            }
-         }
-
-         public static IBusinessRule MaxGreaterThanMin
-         {
-            get
-            {
-               return CreateRule.For<ParameterRange>()
-                  .Property(item => item.MaxValueInDisplayUnit)
-                  .WithRule(maxGreaterThanMin)
-                  .WithError((param, value) => PKSimConstants.Rules.Parameter.MaxGreaterThanMin(param.ParameterName));
-            }
-         }
-
-         public static IBusinessRule MinGreaterThanDbMin
-         {
-            get
-            {
-               return CreateRule.For<ParameterRange>()
-                  .Property(item => item.MinValueInDisplayUnit)
-                  .WithRule(minGreaterThanDbMin)
-                  .WithError((param, value) => PKSimConstants.Rules.Parameter.MinGreaterThanDbMinValue(param.ParameterName, param.DbMinValue, param.Unit.ToString()));
-            }
-         }
-
-         public static IBusinessRule MaxLessThanDbMax
-         {
-            get
-            {
-               return CreateRule.For<ParameterRange>()
-                  .Property(item => item.MaxValueInDisplayUnit)
-                  .WithRule(maxLessThanDbMax)
-                  .WithError((param, value) => PKSimConstants.Rules.Parameter.MaxLessThanDbMaxValue(param.ParameterName, param.DbMaxValue, param.Unit.ToString()));
-            }
-         }
-
-         public static IBusinessRule MinLessThanDbMax
-         {
-            get
-            {
-               return CreateRule.For<ParameterRange>()
-                  .Property(item => item.MinValueInDisplayUnit)
-                  .WithRule(minLessThanDbMax)
-                  .WithError((param, value) => PKSimConstants.Rules.Parameter.MinLessThanDbMaxValue(param.ParameterName, param.DbMaxValue, param.Unit.ToString()));
-            }
-         }
-
-         public static IBusinessRule MaxGreaterThanDbMin
-         {
-            get
-            {
-               return CreateRule.For<ParameterRange>()
-                  .Property(item => item.MaxValueInDisplayUnit)
-                  .WithRule(maxGreaterThanDbMin)
-                  .WithError((param, value) => PKSimConstants.Rules.Parameter.MaxGreaterThanDbMinValue(param.ParameterName, param.DbMinValue, param.Unit.ToString()));
-            }
-         }
+         public static IBusinessRule MinLessThanMax { get; } = CreateRule.For<ParameterRange>()
```

Comment:
@Yuri05 Just creating the variable once instead of every time. much more efficvient